tools/container/hue/hue.sh

```diff
@@ -111,12 +111,28 @@ function set_samlcert() {
   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib64
 }
 
+function task_server_enabled() {
+  for file in "$HUE_CONF_DIR/zhue_safety_valve.ini" "$HUE_CONF_DIR/zhue.ini" "$HUE_CONF_DIR/hue.ini"; do
+    echo "Checking task_server configs in: $file"
+    if grep -A 1 '^\[\[task_server_v2\]\]' "$file" | grep -qi 'enabled=True'; then
+      echo "Task server configs enabled in $file"
+      return 0
+    elif grep -A 1 '^\[\[task_server_v2\]\]' "$file" | grep -qi 'enabled=False'; then
+      echo "Task server configs disabled in $file"
+      return 1
+    else
+      echo "Task server configs not found in $file"
+    fi
+  done
+  return 2
+}
+
 function start_celery() {
   echo "Starting Celery worker..."
   # The schedule of periodic tasks performed by celery-beat is stored in the celerybeat-schedule file.
   touch $HUE_HOME/celerybeat-schedule
   chmod 644 $HUE_HOME/celerybeat-schedule
-  $HUE_BIN/hue runcelery worker --app desktop.celery --loglevel=DEBUG --schedule_file $HUE_HOME/celerybeat-schedule
+  $HUE_BIN/hue runcelery worker --app desktop.celery --loglevel=INFO --schedule_file $HUE_HOME/celerybeat-schedule
```

Comment:
Does this mean it will now only be at INFO level? How can one change it to DEBUG level? Some config or if Hue is set at DEBUG level, will it also pick that up?

Reply:
Added a hue config `celery_log_level` to switch between INFO and DEBUG. Celery starts with INFO and changes based on `celery_log_level` config.